index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3da71c9faf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style/main.css">
+    <script src="./javascript/main.js" defer></script>
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="lato-font">
+    <header id="nav-container">
```

Comment:
- [ ] Kindly add the  top short bar section with social media links, logout, and language links, check the design below 
![image](https://user-images.githubusercontent.com/61408860/169382748-4156c2b2-deee-431d-8fad-248dadaa4bda.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: